M2/Macaulay2/packages/Subring/SubringDoc.m2

```diff
@@ -0,0 +1,374 @@
+-*
+
+   Copyright 2023, Oliver Clarke, Francesca Gandini,
+   Casey Hill, Trevor Karn, Miranda Moore, Chris O'Neill
+    
+   You may redistribute this file under the terms of the GNU General Public
+   License as published by the Free Software Foundation, either version 2 of
+   the License, or any later version.
+*-
+
+
+beginDocumentation()
+
+doc ///
+ Node
+  Key
+   Subrings
```

Comment:
The tests are failing because there's no `Subrings` symbol.

Both the package itself and the main subring class are called `Subring`, but the package should probably be `Subrings` instead as suggested by the documentation.

Renaming it would require:
* Renaming the main package file to `Subrings.m2`
* Updating the entry in the `=distributed-packages` file
* Updated the first argument in the call to `newPackage`